RadarSDK/RadarAPIClient.m

```diff
@@ -503,6 +503,7 @@ - (void)autocompleteQuery:(NSString *)query near:(CLLocation *_Nonnull)near limi
     [queryString appendFormat:@"query=%@", query];
     [queryString appendFormat:@"&near=%.06f,%.06f", near.coordinate.latitude, near.coordinate.longitude];
     [queryString appendFormat:@"&limit=%d", finalLimit];
+    [queryString appendFormat:@"&tags=%@", [layers componentsJoinedByString:@","]];
```

Comment:
May need a `nil` check here

Reply:
nil check now there and using layers not `tag=` 🤦